packages/node_modules/@webex/react-component-confirmation-modal/src/index.js

```diff
@@ -46,7 +54,13 @@ function ConfirmationModal({
 
   return (
     <div>
-      <div className={modalClassNames}>
+      <div
+        ref={modalRef}
```

Comment:
Here I have added an `aria-modal` so it is understood that the window is a modal window. The user then knows how to interact with it.